src/mame/drivers/ksys573.cpp

```diff
@@ -375,6 +376,90 @@ G: gun mania only, drives air soft gun (this game uses real BB bullet)
 
 #define ATAPI_CYCLES_PER_SECTOR ( 5000 )  // plenty of time to allow DMA setup etc.  BIOS requires this be at least 2000, individual games may vary.
 
+// #define JVS_DEBUG
```

Comment:
Please use the stuff from `logmacro.h` for configurable logging.  Especially don’t use `printf` or its friends, they’re not type-safe, and semantics vary depending on the C standard library.

Reply:
Looking into this now and I just noticed the rest of the driver uses `verboselog`. Would it be preferable to just use `verboselog` to keep with the rest of the driver, or would it be better to update the driver's logging code to use `logmacro.h`? I'd rather not mix the two so I can update the entire driver's logging code if `logmacro.h` is the preferable method going forward.

ref: https://github.com/mamedev/mame/blob/983951571ffc1af492d9d944dfaa589b01adac31/src/mame/drivers/ksys573.cpp#L744-L755

Reply:
If you want to work out some categories for the logging that’s already there and convert it, go ahead.

Reply:
I think the `JVS_DEBUG` macro is completely redundant now.

Reply:
I thought I deleted that but it looks like I left the `#define` in. Thanks.